### ISSUE TITLE
Tapping a username or profile icon dosn't do anything

### ISSUE THREAD

**Issue by Transisto** [wireapp/wire-android#2865]
Expected behavior:

Jump to profile or DM.

That's what the desktop version is doing and every single chat app that had groups.

Wire is almost unusable combined with https://github.com/wireapp/wire-android/issues/2858
The only way to DM a user at the moment is to scroll down may many pages to find the user.

**Comment by makingthematrix** [comment]
Thanks for the report. We're working on it.

**Comment by makingthematrix** [comment]
@Transisto : Sorry, but what do you mean by "The only way to DM a user at the moment is to scroll down may many pages to find the user"?

**Comment by Transisto** [comment]
I was wrong, we can find the users we wish to DM by typing their entire
name in the search.
Is that how its supposed to work?

**Comment by makingthematrix** [comment]
@Transisto : I mean, I don't understand what screen are we talking about. The search? The conversation list? The conversation itself?

**Comment by Transisto** [comment]
> Wire is almost unusable when this bug is combined with #2858

Did you read https://github.com/wireapp/wire-android/issues/2858 ?

**Comment by makingthematrix** [comment]
@Transisto :  Ok, so if I understand you correctly, clicking on an entry in the search doesn't open the conversation for you. But I can't reproduce it.

**Comment by Transisto** [comment]
No, I was pointing out that "user search" was broken and tapping a username
in a group chat could have been an alternative to DM this user.

This issue is simply about tapping a user inside a group chat to get to DM.

It works on desktop version but not android.

**Comment by makingthematrix** [comment]
@Transisto : Then that's right. We don't have this feature. If I remember correctly, the decision was that on smaller screens it was too close to the "Like" button and we could have users "liking" messages by mistake.

**Comment by Transisto** [comment]
Feel free to add the "Feature" tag, I've checked the UI and I agree the ❤ is close the the user icon circle, but it's not close the the Username itself.  Tapping the username to get to DM is how most other app works too.
